library/modules/DFSDL.cpp

```diff
@@ -124,3 +134,30 @@ void DFSDL::DFSDL_FreeSurface(SDL_Surface *surface) {
 int DFSDL::DFSDL_PushEvent(SDL_Event *event) {
     return g_SDL_PushEvent(event);
 }
+
+void DFSDL::DFSDL_free(void *ptr) {
+    g_SDL_free(ptr);
+}
+
+char * DFSDL::DFSDL_GetClipboardText() {
+    return g_SDL_GetClipboardText();
+}
+
+int DFSDL::DFSDL_SetClipboardText(const char *text) {
+    return g_SDL_SetClipboardText(text);
+}
+
+DFHACK_EXPORT std::string DFHack::getClipboardTextCp437() {
```

Comment:
Would the `Utf8` wrappers from before still be useful (mainly to handle memory and the std::string conversion)? I suppose if there's no immediate use-case for them, they aren't too hard to add later, but I wouldn't object to adding them now either if you think they're useful.

Reply:
I debated it -- and you can see both versions were in my original commit -- but after reflection, I couldn't think of a use case for the UTF-8 versions and I didn't want to clutter the API